app/services/github_client/pull_request.rb

```diff
@@ -16,10 +16,29 @@ def get
     def files
       url = "repositories/#{repository.github_id}/pulls/#{pull_request.number}/files"
       get_all_paginated_items(url, MAX_FILES_PER_PAGE)
+    rescue Faraday::Error => exception
+      handle_exception(exception)
+      []
+    end
+
+    def comments
+      url = "repos/#{repository.full_name}/pulls/#{pull_request.number}/comments"
+      get_all_paginated_items(url, MAX_FILES_PER_PAGE)
+    rescue Faraday::Error => exception
+      handle_exception(exception)
+      []
     end
 
     private
 
     attr_reader :pull_request
+
+    def handle_exception(exception)
+      Honeybadger.notify(exception)
```

Comment:
I added Honeybadger because it is the tool being used, but I don't have access to it. I think in a future PR, we can move to Sentry or something similar. I added it anyway to keep track of this error notification.